src/routes/feedback/components/FeedbackBox.js

```diff
@@ -0,0 +1,205 @@
+import type { FeedbackDataType } from '../../../modules/endpoint/FeedbackEndpoint'
+import { CATEGORIES_ROUTE } from '../../../modules/app/routes/categories'
+import { EVENTS_ROUTE } from '../../../modules/app/routes/events'
+import { EXTRAS_ROUTE } from '../../../modules/app/routes/extras'
+import { SEARCH_ROUTE } from '../../../modules/app/routes/search'
+import { DISCLAIMER_ROUTE } from '../../../modules/app/routes/disclaimer'
+import FeedbackHeader from './FeedbackHeader'
+import FeedbackComment from './FeedbackComment'
+import type { LocationState } from 'redux-first-router'
+import { goToFeedback } from '../../../modules/app/routes/feedback'
+import FeedbackDropdownItem from '../FeedbackDropdownItem'
+
+const StyledFeedbackBox = styled.div`
+  display: flex;
+  width: 400px;
+  height: auto;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+  border-radius: 10px;
+  border-color: #585858;
+  font-size: ${props => props.theme.fonts.contentFontSize};
+`
+
+export const Description = styled.div`
+  padding: 10px 0 5px;
+`
+
+const SubmitButton = styled(CleanLink)`
+  margin: 15px 0;
+  padding: 5px;
+  background-color: ${props => props.theme.colors.themeColor};
+  color: ${props => props.theme.colors.backgroundAccentColor};
+  text-align: center;
+  border-radius: 0.25em;
+`
+
+type PropsType = {
+  cities: Array<CityModel>,
+  title?: string,
+  alias?: string,
+  id?: number,
+  query?: string,
+  isPositiveRatingSelected: boolean,
+  location: LocationState,
+  isOpen: boolean,
+  commentMessageOverride: ?string,
+  hideHeader: boolean,
+  t: TFunction
+}
+
+type StateType = {
+  feedbackOptions: Array<FeedbackDropdownItem>,
+  selectedFeedbackOption: FeedbackDropdownItem,
+  comment: string
+}
+
+export class FeedbackBox extends React.Component<PropsType, StateType> {
+  static defaultProps = {
+    hideHeader: false
+  }
+  constructor (props: PropsType) {
+    super(props)
+    const feedbackOptions = this.getFeedbackOptions()
+    this.state = {feedbackOptions: feedbackOptions, selectedFeedbackOption: feedbackOptions[0], comment: ''}
+  }
+
+  onCommentChanged = (event: SyntheticInputEvent<HTMLTextAreaElement>) => this.setState({comment: event.target.value})
+
+  onFeedbackOptionChanged = (selectedDropdown: FeedbackDropdownItem) => {
+    this.setState({selectedFeedbackOption: selectedDropdown})
+  }
+
+  componentDidUpdate (prevProps: PropsType) {
+    const {isOpen} = this.props
+    const prevIsOpen = prevProps.isOpen
+
+    // If the FeedbackBox is opened, we have to reset and initialize the state
+    if (prevIsOpen !== isOpen && isOpen) {
+      /* eslint-disable react/no-did-update-set-state */
+      const feedbackOptions = this.getFeedbackOptions()
+      const selectedFeedbackOption = feedbackOptions[0]
+      this.setState({feedbackOptions: feedbackOptions, selectedFeedbackOption: feedbackOptions[0], comment: ''})
```

Comment:
Why do you need to copy the props into the state here?

Reply:
This resets the feedback options to choose from as well as the entered comment. Otherwise it would stay the same no matter where you open the feedback box from. Which means if you open the feedback box once in /augsburg/de and the next time in /augsburg/de/willkommen, you will not have the option "Inhalt der Seite Willkommen" and the same comment prefilled which you typed for /augsburg/de. The reason for this is that the FeedbackBox always lives in the background.